app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -14,6 +14,7 @@ model: ProvisioningTemplate
 <%= "\n# Location: [#{@location.name}]" if @location -%>
 <%= "\n# Hostgroup: [#{@hostgroup.name}]" if @hostgroup -%>
 <%= "\n# Operating System: [#{@operatingsystem.name}]" if @operatingsystem -%>
+<%= "\n# Setup Insights: [#{@setup_insights}]" if !@setup_insights.nil? -%>
```

Comment:
`unless @setup_insights.nil?` would be slightly more readable, the rule of thumb for me is, if there's just one argument, it's ok, if there's operator such as `==` or `>`, I prefer if. The same applies to the two below.